unity-renderer/Assets/Tutorial/Scripts/TutorialController.cs

```diff
@@ -580,14 +527,19 @@ private void MoreMenu_OnRestartTutorial()
 
         private bool IsPlayerInsideGenesisPlaza()
```

Comment:
I think this method can be made pure and `static` and be unit tested if we just inject the `world` and needed dependencies.